modules/websocketpp/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "homepage": "https://github.com/zaphoyd/websocketpp",
+  "maintainers": [
+    {
+      "email": "bcr-maintainers@bazel.build",
+      "name": "No Maintainer Specified"
```

Comment:
@lalten Do you mind adding yourself as a maintainer of this dependency?

Reply:
Sorry, I don't think I'm the right one for this. The upstream project seems to be unmaintained by the author @zaphoyd. The BUILD was written by @mvukov in https://github.com/mvukov/rules_ros2/blob/main/repositories/websocketpp.BUILD.bazel.

Reply:
OK, thanks for the contribution!